File: megumi/src/main/java/com/erisu/cloud/megumi/util/RedisKey.java
package com.erisu.cloud.megumi.util;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Getter
public enum RedisKey {
    PLUGIN("plugin");
    private String name;

    RedisKey(String name) {
        this.name = name;
    }
}

File: megumi/src/main/java/com/erisu/cloud/megumi/util/RedisKey.kt
package com.erisu.cloud.megumi.util

/**
 *@Description TODO
 *@Author alice
 *@Date 2021/6/23 8:33
 **/
enum class RedisKey(val key: String) {
    PRINCESS_NAME("princess:name"), PLUGIN("plugin"),GACHE("gacha"),PIXIV("pixiv"),BILIBILI("bilibili"),BOSS_ORDER("boss:order"),
    BOSS_IN("boss:in"),OSU_BG("osu:bg"),OSU_BG_WINNER("osu:bg:winner")
}